e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-to-an-existing-payment.spec.js

```diff
@@ -12,22 +12,25 @@ import { NODE_TASKS } from '../../../../../e2e-fixtures';
 import relative from '../../relativeURL';
 
 context(`${PDC_TEAMS.PDC_RECONCILE} users can add fee records to existing payments`, () => {
+  const PAYMENT_ID_ONE = '1';
```

Comment:
If there is no second payment this could just be `PAYMENT_ID`

Reply:
Kept this suffix as we now have a second payment (to test that multiple payment groups can be rendered)